blank-canvas-blocks/assets/navigation.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * File navigation.js.
+ *
+ * Required to open the mobile navigation.
+ */
+(function () {
+
+	function addMobileMenuButton( navMenu ) {
+
+		const menuContainer = navMenu.querySelector( '.wp-block-navigation__container' );
+		const openButton = document.createElement( 'a' );
+		const openButtonLabel = getComputedStyle(menuContainer).getPropertyValue( '--wp--custom--navigation--mobile--menu--label' );
+		openButton.setAttribute( 'href', '#' );
+		openButton.classList.add( 'wp-block-navigation__mobile-menu-open-button' );
+		openButton.innerText = ( openButtonLabel );
+		openButton.addEventListener( 'click', () => {
+			navMenu.classList.add( 'show' );
+			menuContainer.scrollTop = 0;
+		});
+
+		menuContainer.addEventListener( 'click', (mouseEvent) => {
```

Comment:
For accessibility reasons, I think there needs to be an explicit close button, rather than clicking anywhere in the container to close it. This way, it can also be closed via keyboard navigation.

Reply:
Good call.  I had wanted to avoid that to play golf with the amount of JavaScript but you're right there.

Reply:
Close **button** added.